src/Common/Compat.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
sure I can do that, do you mean adding the note on the PR description? or do you actually want me to add an .md file on how to create cs from a .dll using GenAPI?